src/screens/MessageScreen.tsx

```diff
@@ -122,8 +122,7 @@ export function MessageScreen<ParamList extends ParamListBase>({
           <View
             style={{
               flexDirection: 'row',
-              flex: 1,
-              maxWidth: 60,
+              width: 60,
```

Comment:
After the upgrade the message avatars did not display correctly. Setting a static width solved this problem.